elixir/apps/domain/lib/domain/auth/adapter/openid_connect/directory_sync.ex

```diff
@@ -140,12 +140,12 @@ defmodule Domain.Auth.Adapter.OpenIDConnect.DirectorySync do
     )
 
     finish_time = System.monotonic_time(:millisecond)
-    Logger.info("Finished syncing in #{time_taken(start_time, finish_time)}")
+    Logger.info("Finished syncing #{adapter} providers in #{time_taken(start_time, finish_time)}")
   end
 
   defp sync_provider(module, provider) do
     start_time = System.monotonic_time(:millisecond)
-    Logger.debug("Syncing provider")
+    Logger.debug("Syncing provider: #{provider.id}")
```

Comment:
I believe `debug` isn't enabled on prod, might be good to have `info` level here?

Reply:
Sounds good to me.  This was already set to `debug`, I just changed the actual message, but I'm happy to move this to `info` if you'd like.